File: program/convert-string-to-pathcase/convert-string-to-pathcase.java
class replaceSpace
{
	static String replaceSpace(String str)
	{
		String s = "";
		for (int i = 0; i < str.length(); ++i) {
			if (str.charAt(i) == ' ')
				s += '/';
			else
				s += str.charAt(i);
		}
		return s;
	}
	public static void main(String []args)
	{
	    Scanner sc= new Scanner(System.in);
        System.out.print("Enter a string: ");  
        String str= sc.nextLine();   
		System.out.println(replaceSpace(str));
	}
}

File: program/program/convert-string-to-pathcase/ConvertStringToPathcase.kt
import java.util.Scanner

fun main() {
    print("Enter String : ")
    println()
    var min = readLine()

    println("Camel Case: " + ConvertStringToPathcase(min!!.split(" ")))

}

private fun ConvertStringToPathcase(args: List<String>): String {
    val buffer = StringBuffer()
    for((index,day) in args.withIndex()){
        //println("$index :$day")
        if(index == 0) {
            buffer.append(day);
        } else {
            buffer.append("/").append(day)
        }
    }

    return buffer.toString()
}